src/components/WorkQueueDetails.vue

```diff
@@ -67,7 +68,22 @@
     return num > 0 ? `${num} Deployments` : 'Deployments'
   })
 
+  const api = useWorkspaceApi()
+
   const { workQueueStatus } = useWorkQueueStatus(props.workQueue.id)
+  const workPoolArgs = computed<Parameters<typeof api.workPools.getWorkPools> | null>(() => {
```

Comment:
```suggestion
  const workPoolArgs = computed<[WorkPoolsFilter] | null>(() => {
```
We should use the filter type here - it's more explicit and easier to read